BUILD.bazel

```diff
@@ -981,24 +938,263 @@ ray_cc_library(
+        "@com_google_absl//absl/container:flat_hash_map",
         "@com_google_absl//absl/container:flat_hash_set",
-        "@com_google_absl//absl/memory",
+    ],
+)
+
+ray_cc_library(
+    name = "cluster_task_manager_interface",
+    hdrs = ["src/ray/raylet/scheduling/cluster_task_manager_interface.h"],
+    deps = [
+        ":rpc_server_call",
+        "//src/ray/protobuf:node_manager_cc_proto",
+    ],
+)
+
+ray_cc_library(
+    name = "local_task_manager_interface",
+    hdrs = ["src/ray/raylet/scheduling/local_task_manager_interface.h"],
+    deps = [
+        ":ray_common",
+        ":scheduler_internal",
+        "@com_google_absl//absl/container:flat_hash_map",
+        "@com_google_absl//absl/container:flat_hash_set",
+    ],
+)
+
+ray_cc_library(
+    name = "local_resource_manager",
+    srcs = ["src/ray/raylet/scheduling/local_resource_manager.cc"],
+    hdrs = ["src/ray/raylet/scheduling/local_resource_manager.h"],
+    deps = [
+        ":gcs_client_lib",
+        ":ray_common",
+        "//src/ray/protobuf:gcs_cc_proto",
+        "//src/ray/protobuf:node_manager_cc_proto",
+        "//src/ray/util:logging",
+        "@com_google_absl//absl/container:flat_hash_map",
+        "@com_google_absl//absl/container:flat_hash_set",
+        "@com_google_googletest//:gtest_prod",
+    ],
+)
+
+ray_cc_library(
+    name = "affinity_with_bundle_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/affinity_with_bundle_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/affinity_with_bundle_scheduling_policy.h"],
+    deps = [
+        ":ray_common",
+        ":scheduling_policy",
+    ],
+)
+
+ray_cc_library(
+    name = "bundle_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/bundle_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/bundle_scheduling_policy.h"],
+    deps = [
+        ":cluster_resource_manager",
+        ":ray_common",
+        ":scheduling_context",
+        ":scheduling_policy",
+        ":scorer",
+    ],
+)
+
+ray_cc_library(
+    name = "composite_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/composite_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/composite_scheduling_policy.h"],
+    deps = [
+        ":affinity_with_bundle_scheduling_policy",
+        ":bundle_scheduling_policy",
+        ":cluster_resource_manager",
+        ":hybrid_scheduling_policy",
+        ":node_affinity_scheduling_policy",
+        ":node_label_scheduling_policy",
+        ":random_scheduling_policy",
+        ":spread_scheduling_policy",
+        "//src/ray/util",
+        "//src/ray/util:container_util",
+    ],
+)
+
+ray_cc_library(
+    name = "hybrid_scheduling_policy",
+    srcs = ["src/ray/raylet/scheduling/policy/hybrid_scheduling_policy.cc"],
+    hdrs = ["src/ray/raylet/scheduling/policy/hybrid_scheduling_policy.h"],
+    deps = [
+        ":scheduling_policy",
+        "//src/ray/util",
```

Comment:
Same here, could you please remove the header and deps for `ray/util`?
https://github.com/ray-project/ray/blob/34ae461121a8d952c941833a3df0e37900aeb82f/src/ray/raylet/scheduling/policy/hybrid_scheduling_policy.cc#L20